packages/cli/src/controllers/users.controller.ts

```diff
@@ -294,11 +289,11 @@ export class UsersController {
 
 		await this.userService.update(targetUser.id, { role: payload.newRoleName });
 
-		this.internalHooks.onUserRoleChange({
-			user: req.user,
-			target_user_id: targetUser.id,
-			target_user_new_role: ['global', payload.newRoleName].join(' '),
```

Comment:
Pre-existing bug: The target user's new role is being reported e.g. as 'global global:member'. To be fixed separately.

Reply:
I'll follow up on this, won't forget